### DISCUSSION TITLE
How to change lockscreen image before startup!

### DISCUSSION THREAD

**Discussion by ghost** [t1m0thyj/WinDynamicDesktop#393] [General]
How to change lockscreen image before startup!
i tried alot of things it is stuck to night wallpaper of miami(even when deleted)

**Comment by t1m0thyj** [comment]
This is a known issue, see https://github.com/t1m0thyj/WDD-scripts/issues/1 I'd like to fix it, see the pinned issue: https://github.com/t1m0thyj/WinDynamicDesktop/issues/321

But so far I have not been able to find a way to overcome the issue, hence why the script is marked as experimental for now.

**Reply by t1m0thyj** [reply]
Since WDD 5.0, changing the lockscreen image is included in the core app and should function reliably.
